lp-builder-config/ceph.yaml

```diff
@@ -126,6 +126,78 @@ projects:
     charmhub: ceph-fs
     launchpad: charm-ceph-fs
     repository: https://opendev.org/openstack/charm-ceph-fs.git
+    branches:
+      master:
+        build-channels:
+          charmcraft: "2.0/stable"
+        channels:
+          - latest/edge
+        bases:
+          - "20.04"
+          - "22.04"
+          - "22.10"
+    stable/quincy.2:
+      build-channels:
+        charmcraft: "1.5/stable"
+      channels:
+        - quincy/stable
```

Comment:
Sorry, one final issue; although this is now copied, due to indentation requirements, this section from here down (the branches from `stable/quincy.2` onwards) needs to be indented by 2 more spaces.  This is unfortunately because the default set is less indented than per charm sets of branches.  Sorry!